webapp/src/components/Staking/StakingOverview.tsx

```diff
@@ -97,6 +103,40 @@ const OverviewLabel = styled(SecondaryText)`
 `;
 
 const StakingOverview = () => {
+  const { stakingPools, loading: stakingLoading } = useStakingPool(VaultList);
+  const { data: tokenData, loading: tokenLoading } = useRBNToken();
+  const loadingText = useTextAnimation(
+    ["Loading", "Loading .", "Loading ..", "Loading ..."],
+    250,
+    stakingLoading || tokenLoading
+  );
+
+  const totalRewardDistributed = useMemo(() => {
+    if (stakingLoading) {
+      return loadingText;
+    }
+
+    let totalDistributed = BigNumber.from(0);
+
+    for (let i = 0; i < VaultList.length; i++) {
+      const stakingPool = stakingPools[VaultList[i]];
+      if (!stakingPool) {
+        break;
+      }
+      totalDistributed = totalDistributed.add(stakingPool.totalRewardClaimed);
+    }
+
+    return formatBigNumber(totalDistributed, 2, 18);
+  }, [stakingLoading, loadingText, stakingPools]);
```

Comment:
Something to discuss with Vinnie- since rewards can only be claimed at the end of the 4 week rewards duration, this number will be 0 most of the time. Should we remove this metric?

Reply:
Sure